### PR TITLE
fix android enter bug

### PR DESCRIPTION
**Pull request by justame**

### CODE REVIEW

**Comment by justame** [comment]
@lxgreen I don't have a permission to Percy, can you take a look and let me know if it's safe to merge?

**Comment by oded-soffrin** [comment]
Fixed on surge in my android 🤩🎉🎊

File: tsconfig.json [hunk 1]
```diff
@@ -10,7 +13,11 @@
     "moduleResolution": "node",
     "target": "ESNext",
     "jsx": "react",
-    "lib": ["dom", "ESNext"],
+    "lib": [
+      "dom",
+      "ESNext"
+    ],
+    "skipLibCheck": true,
```
Comment (lxgreen) [inline tsconfig.json:20]: i'd limit this flag to specific packages, using the tsconfig extensibility. let's talk

**Review by lxgreen — CHANGES_REQUESTED** [review]

**Comment by lxgreen** [comment]
@yaronn the resolver unit tests fail. this points to incompatibility we have to handle. `skipLibCheck` is dangerous way to go -- we just replace dependency versions and ignore type errors. The type incompatibility usually points to API incompatibility, and we don't know how many obscure bugs added.

**Pull request closed without merging**